packages/ag-charts-community/src/chart/dom/domManager.ts

```diff
@@ -13,38 +14,48 @@ const domElementConfig: Record<
     styles: { childElementType: 'style' },
     canvas: { childElementType: 'canvas' },
     'canvas-overlay': { childElementType: 'div' },
-    hidden: { childElementType: 'div' },
 };
 
 const STYLES = `
-.ag-chart-wrapper {
+.ag-charts-wrapper {
+    position: relative;
     touch-action: none;
     box-sizing: border-box;
-    overflow: hidden;
+    display: flex !important;
+    align-items: center !important;
+    justify-content: center !important;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
```

Comment:
This does nothing for position: relative

Reply:
I'll remove.